mutable-copy-runtime/src/main/kotlin/fp/serrano/MutableCopy.kt

```diff
@@ -0,0 +1,5 @@
+package fp.serrano
+
+@Target(AnnotationTarget.CLASS)
+@Retention(AnnotationRetention.SOURCE)
+public annotation class MutableCopy
```

Comment:
Could you make this part of `transformative-types`? I don't think having so many project helps here.

Reply:
Also, we should agree on either using lowercase (`@transformative`) or camelcase (`@MutableCopy`)

Reply:
I'll move it :+1: :slightly\_smiling\_face:   
I would personally go with Kotlin's standard :sweat\_smile: 

![](https://user-images.githubusercontent.com/260697/188738822-95d725ba-aa36-4f03-b314-85147bd1d4b0.png)

Also, I think there is room for having a single annotation. I've created an issue with some ideas regarding how we could simplify the API.

I'm happy to keep \`@Transformative\`   if we decide so :slightly\_smiling\_face: 

Hopefully, we can evolve this to not require an annotation one day :crossed\_fingers: